assets/colorcet.m

```diff
@@ -9073,7 +9073,7 @@
 
 case { 'C1'  'C01' }
 descriptorname = 'cyclic_mrybm_35-75_c68_n256';
-description = 'Cyclic: magenta - red - yellow - blue. Alows four orientations/phase angles to be visulaised.';
+description = 'Cyclic: magenta - red - yellow - blue. Allows four orientations/phase angles to be visulaised.';
```

Comment:
Please remove any fixes to the contents of `assets`; these files are mirrors of upstream content whose source is located elsewhere, and we do not want those files to diverge from their source files. Ideally such fixes would be filed at the upstream repo, though I'm not sure where that is (we get the files from https://colorcet.com).